root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4405,6 +4405,16 @@ limited_link_type_combinations: [
              input_entity_type: 'release',
     expected_relationship_type: 'downloadpurchase',
   },
```

Comment:
Following https://github.com/metabrainz/musicbrainz-server/pull/3132/files#r1444824078, here is an applicable suggestion to test validation in the already existing test for release from OTOTOY.

```suggestion
       only_valid_entity_types: ['release'],
  },
```